stories/features/rewards/concepts.tsx

```diff
@@ -51,7 +51,10 @@ const defaultGrant = {
   probi: '',
   expiryTime: 0,
   captcha: '',
-  hint: ''
+  hint: '',
+  finishTitle: 'It\'s your lucky day!',
+  finishText: 'Your token grant is on its way',
```

Comment:
needs a full-stop to be consistent with UGP

Reply:
done